packages/tools/devtools/devtools-core/src/IFluidDevtools.ts

```diff
@@ -33,6 +34,15 @@ export interface IFluidDevtools extends IDisposable {
 	 */
 	registerContainerDevtools(props: ContainerDevtoolsProps): void;
 
+	/**
+	 * Registers a container runtime with the devtools.
+	 *
+	 * @remarks To remove the Container Runtime from the Devtools, call {@link IFluidDevtools.closeContainerDevtools}.
+	 *
+	 * @throws Will throw if devtools have already been registered for the specified {@link ContainerKey}.
+	 */
+	registerContainerRuntimeDevtools(props: ContainerRuntimeDevtoolsProps): Promise<void>;
```

Comment:
Nit: I'm not super happy with the current "registerContainerDevtools" method naming. The user isn't giving us a devtools instance; they're giving us a _container_. I think I would prefer we call this `registerContainerRuntime`, and in the future, we can rename `registerContainerDevtools` to `registerContainer`.